src/err.rs

```diff
@@ -56,7 +57,21 @@ pub struct PyErr {
 pub type PyResult<T> = Result<T, PyErr>;
 
 /// Marker type that indicates an error while downcasting
-pub struct PyDowncastError;
+#[derive(Debug)]
+pub struct PyDowncastError<'a, 'b> {
+    from: &'a PyAny,
+    to: Cow<'b, str>,
+}
+
+impl<'a, 'b> PyDowncastError<'a, 'b> {
+    pub fn new(from: &'a PyAny, to: Cow<'b, str>) -> Self {
+        PyDowncastError { from, to }
+    }
+
+    pub fn set_to(&mut self, to: Cow<'b, str>) {
```

Comment:
What is this method for?

Reply:
That's an accidental leftover from trying to get the `Optional` info to work, I'll remove it!